examples/mapping-with-cue/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
```

Comment:
Let's not call this kustomization but rather something like sample_kustomization.yaml. As we've seen earlier kustomization.yaml is special.

Reply:
excellent point. i've renamed all of the examples.